### DISCUSSION TITLE
Бюро находок в мобильном приложении

### DISCUSSION THREAD

**Discussion by 0niel** [0niel/university-app#224] [Идеи]
Можно добавить небольшой сервис, который будет называться "Бюро находок". В нём люди могут оставлять записи с фотографиями и описанием, если они что-то потеряли или что-то нашли в вузе. Будет фильтр по дате добавления, а также по тегам: актуально/неактуально, нашёл/потерял.

**Comment by ValeryVerkhoturov** [comment]
Необходимо продумать методы цензурирования текста и фото.

**Reply by Dragonprod** [reply]
Потеряшки должны проходить модерацию перед попаданием в общий каталог

**Reply by 0niel** [reply]
> Потеряшки должны проходить модерацию перед попаданием в общий каталог

Поддерживаю эту идею. Для всех изменений и новых записей можно делать модерацию, это проще чем цензурирование
